config/gcr-tekton-dashboard.yaml

```diff
@@ -0,0 +1,59 @@
+# ------------------- Dashboard Deployment ------------------- #
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tekton-dashboard
+  labels:
+    app: tekton-dashboard
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: tekton-dashboard
+  template:
+    metadata:
+      name: tekton-dashboard
+      labels:
+        app: tekton-dashboard
+    spec:
+      containers:
+      - name: tekton-dashboard
+        image: gcr.io/tekton-nightly/dashboard:latest
+        ports:
+        - containerPort: 9097
+        livenessProbe:
+          httpGet:
+            path: /health
+            port: 9097
+        readinessProbe:
+          httpGet:
+            path: /readiness
+            port: 9097
+        resources:
+        env:
+        - name: PORT
+          value: "9097"
+        - name: WEB_RESOURCES_DIR
+          value: /var/run/ko/web
+        - name: PIPELINE_RUN_SERVICE_ACCOUNT
+          value: ""
+        - name: INSTALLED_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+---
+# ------------------- Dashboard Service ------------------- #
+kind: Service
```

Comment:
Hi @vtereso do we need this service? The docs guide users to access the dashboard via `kubectl proxy`.

Reply:
The `kubectl proxy` command proxies requests over the api server to kubernetes resources, in this case our service. The Kubernetes dashboard uses the same convention as Steven noted in the original change to use the clusterIP service rather than nodePort.

Reply:
As I understand it, the request goes through the api server to the kube proxy such that the additional forwarding must be resolved by a service.

Reply:
ok great thank you!